enterprise/server/workflow/service/service.go

```diff
@@ -821,6 +821,11 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 	if workflowAction.SelfHosted {
 		useSelfHostedExecutors = "true"
 	}
+	pool := ws.WorkflowsPoolName()
+	if workflowAction.Pool != "" {
+		pool = workflowAction.Pool
+	}
```

Comment:
I don't think we'll need to configure the pool for the workflow action, since the CI_runner can still run on the normal workflow pool - the RBE action that it executes just needs to be run on the bare pool.

You can set `exec_properties = {"Pool": "bare"}` on `firecracker_test` to get that to work (also with the new approach you'll need `"use-self-hosted-executors": "true"`